### PR TITLE
Create subtask for model and create method #9

### PR DESCRIPTION
**Pull request by dinhthi12**
# Related Tickets

- [#Ticket_ID](https://edu-redmine.vn/issues/xxxx)

## WHAT this PR does?

- ex: Change number items `completed/total` in admin page.

## Notes Impacted Areas

- N/A.

## Attachments

- N/A.

### CODE REVIEW

**Comment by dinhthi12** [comment]
ready

File: db/seeds.rb [hunk 1]
```diff
@@ -27,6 +27,6 @@
     title: Faker::Lorem.sentence,
     description: Faker::Lorem.paragraph,
     due_date: Faker::Date.forward(days: 30),
-    completed: [true, false].sample
+    completed: rand(3)
   )
 end
```
Comment (tn0810520) [inline db/seeds.rb:32]: Thêm seed subtask

File: db/schema.rb [hunk 2]
```diff
@@ -10,16 +10,27 @@
...
+    t.datetime "updated_at", null: false
+    t.bigint "task_id"
+    t.index ["task_id"], name: "index_sub_tasks_on_task_id"
+  end
+
   create_table "tasks", force: :cascade do |t|
     t.string "title"
     t.text "description"
     t.date "due_date"
-    t.boolean "completed"
```
Comment (tn0810520) [inline db/schema.rb:22]: OK

File: app/models/task.rb [hunk 3]
```diff
@@ -1,3 +1,6 @@
 class Task < ApplicationRecord
+  enum completed: { not_started: 0, in_progress: 1, completed: 2 }
+
   belongs_to :user
+  has_many :sub_tasks, dependent: :destroy
 end
```
Comment (hoangpt1310) [inline app/models/task.rb:6]: OK

**Comment by hoangpt1310** [comment]
merged

**Pull request merged**